Examples/Framework/include/ActsExamples/Framework/WriterT.hpp

```diff
@@ -16,9 +16,13 @@
 #include "ActsExamples/Framework/WhiteBoard.hpp"
 #include <Acts/Utilities/Logger.hpp>
 
+#include <limits>
 #include <memory>
 #include <string>
 
+#define NaNdouble std::numeric_limits<double>::quiet_NaN()
+#define NaNfloat std::numeric_limits<float>::quiet_NaN()
```

Comment:
Hm, could you maybe switch this to something like 

```suggestion
constexpr double NaNdouble std::numeric_limits<double>::quiet_NaN();
constexpr float NaNfloat std::numeric_limits<float>::quiet_NaN();
```
which I believe should work. Not absolutely critical, but prevents the use of the preprocessor for this. What to you think?

Reply:
Sounds good! Fixed.